webapp/channels/src/components/common/hooks/useGetTotalUsersNoBots.ts

```diff
@@ -10,12 +10,12 @@ const useGetTotalUsersNoBots = (includeInactive = false): number => {
     const dispatch = useDispatch();
     const [userCount, setUserCount] = useState<number>(0);
 
-    const getTotalUsers = async () => {
-        const {data} = await dispatch(getFilteredUsersStats({include_bots: false, include_deleted: includeInactive}, false));
-        setUserCount(data?.total_users_count ?? 0);
-    };
-
     useEffect(() => {
+        const getTotalUsers = async () => {
+            const {data} = await dispatch(getFilteredUsersStats({include_bots: false, include_deleted: includeInactive}, false));
+            setUserCount(data?.total_users_count ?? 0);
+        };
+
         getTotalUsers();
     }, []);
```

Comment:
Effects work on a different basis than callbacks and memos, and I didn't want to change the behavior.

For example, this effect is used as a "onMount". On mount, we want to get the users. We don't want to do that whenever any of the props changes, just on mount.

It could be argued that we should do this whenever `includeInactive` changes, but I did not want to mess with the current behavior.